app/controllers/admin/users_controller.rb

```diff
@@ -70,10 +73,14 @@ def update
         return
       end
 
-      existing_health_roles = @user.roles.health.to_a
+      existing_health_roles = @user.health_roles.to_a # TODO: START_ACL remove when ACL transition complete
```

Comment:
Are we going to try and migrate the health permissions? I thought we had said no...

Reply:
Correct, that looks like a spurious comment